indexing-service/src/test/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTaskTest.java

```diff
@@ -166,16 +164,19 @@ public void testKillWithMarkUnused() throws Exception
             null
         );
 
-    Assert.assertEquals(ImmutableList.of(newSegment(Intervals.of("2019-02-01/2019-03-01"), version)), unusedSegments);
+    Assert.assertEquals(ImmutableList.of(segment2), unusedSegments);
     Assertions.assertThat(
-        getMetadataStorageCoordinator()
-            .retrieveUsedSegmentsForInterval(DATA_SOURCE, Intervals.of("2019/2020"), Segments.ONLY_VISIBLE)
-    ).containsExactlyInAnyOrder(
-        newSegment(Intervals.of("2019-01-01/2019-02-01"), version),
-        newSegment(Intervals.of("2019-04-01/2019-05-01"), version)
-    );
+        getMetadataStorageCoordinator().retrieveUsedSegmentsForInterval(
+            DATA_SOURCE,
+            Intervals.of("2019/2020"),
+            Segments.ONLY_VISIBLE
+        )
+    ).containsExactlyInAnyOrder(segment1, segment4);
 
-    Assert.assertEquals(new KillTaskReport.Stats(1, 2, 1), getReportedStats());
```

Comment:
Doesn't seem like it needs to be broken up across multiple lines. Seems readable as well as compact the way it currently is. (same comment in a couple other places)

Reply:
Yeah, it doesn't seem like much when looking at this line, but with the IDE providing parameter hints, things seem longer than they appear in a split screen setting. So I often end up scrolling right through the screen and the line break helps with that.